src/osdc/Objecter.h

```diff
@@ -2446,7 +2446,9 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 				 boost::system::error_code>>> waiting_for_map;
 
   ceph::timespan mon_timeout;
+  bool mon_timeout_configured = false;
```

Comment:
Code keeps changing since my last review -- but these are no longer used in your latest version.